algosdk/transaction.py

```diff
@@ -52,11 +58,19 @@ def __init__(
 
     def get_txid(self):
         """
+        NOTE: This method is deprecated:
+        Please use the v2 equivalent in `future.transaction` instead.
+
         Get the transaction's ID.
 
         Returns:
             str: transaction ID
         """
+        warnings.warn(
```

Comment:
Adding warnings to each method might be pedantic - in `algod.py`, only the constructor has a `DeprecationWarning` triggered.

Reply:
I agree warnings in each method is likely too much. If there's a warning in the constructor, that should be good enough in my opinion